start.rst

```diff
@@ -9,15 +9,15 @@ NorESM2 User's Guide
 This guide instructs both novice and experienced users on building and running NorESM2 for various experiment set ups. The chapters attempt to provide relatively detailed information on how to make, set up, build, run and modify experiments using NorESM2.
 
 
-NorESM2.1
+NorESM2.3
 ^^^^^^^^^
-NorESM2.1 (**NorESM tag release-noresm2.1.1**) is a technical development version of NorESM2 that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (**NorESM tag release-noresm2.0.7**). NorESM2.1 relies on the same case creation (including compsets), set up, build and run procedures as NorESM2.0, but will not generally produce the same model output. We will refer to NorESM2.1 as "NorESM2" throughout this documentation.
+NorESM2.3 (**NorESM tag release-noresm2.3.?**) is a technical development version of NorESM2 that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (**NorESM tag release-noresm2.0.7**). NorESM2.3 relies on the same case creation (including compsets), set up, build and run procedures as NorESM2.0, but will not generally produce the same model output. We will refer to NorESM2.3 as "NorESM2" throughout this documentation.
```

Comment:
Should we define NorESM2.3 relative to NorESM2.1?
```suggestion
NorESM2.3 (**NorESM tag release-noresm2.3.?**) is a technical development version of NorESM2 that includes a few bug fixes along with numerous code improvements from the NorESM2.1 release (**NorESM tag release-noresm2.1.1**). NorESM2.3 relies on the same case creation (including compsets), set up, build and run procedures as NorESM2.0, but will not generally produce the same model output. We will refer to NorESM2.3 as "NorESM2" throughout this documentation.
```

Reply:
Yes, I think it makes sense to use NorESM2.1 as reference. We may want to update this text later on, but I agree we can put in the NorESM2.1 reference already now.